client/src/components/settings/email.tsx

```diff
@@ -173,7 +173,12 @@ function EmailSettings({
         </FullWidthRow>
       )}
       <FullWidthRow>
-        <form id='form-update-email' onSubmit={handleSubmit}>
+        <form
+          id='form-update-email'
+          {...(!isDisabled
+            ? { onSubmit: handleSubmit }
+            : { onSubmit: e => e.preventDefault() })}
```

Comment:
Needed to do it this way, because handleSubmit was used in Quincy email subscription